changelog.md

```diff
@@ -8,6 +8,7 @@
 * Add file descriptors resolver callback based on content URI - Unix platform only
 * Fix database foreign keys not being enabled
 * Remove `ServiceStop` error
+* Disallow file cancels from the sending side
```

Comment:
Is this right?

Reply:
It's not a feature of this PR but wanted to sneak that in.
It's true, automated retires do not allow canceling files from the sender, we discussed it yesterday on Slack

Reply:
ah okay, wasn't sure since it wasn't related to the PR 👍